compiler/src/dotty/tools/dotc/CompilationUnit.scala

```diff
@@ -105,7 +106,7 @@ class CompilationUnit protected (val source: SourceFile) {
 
 object CompilationUnit {
 
-  class SuspendException extends Exception
+  class SuspendException extends ControlThrowable
```

Comment:
This change came up recently and was rejected: https://github.com/lampepfl/dotty/pull/16616#discussion_r1062295350
More generally it seems from https://contributors.scala-lang.org/t/nonfatal-and-controlthrowable-changed-in-stdlib/6049 that there is a desire to move away / deprecate ControlThrowable.